arch/arm/boot/dts/overlays/README

```diff
@@ -1045,10 +1045,32 @@ Load:   dtoverlay=hifiberry-amp
 Params: <None>
 
 
-Name:   hifiberry-dac
```

Comment:
I don't think you meant to delete this entry.

Reply:
Thanks for finding! I guess that came through patching it from an older repo into a newer one. I've done the changes in November last year when we have field tested everything in HBOS.